src/sw/redis++/async_redis.h

```diff
@@ -372,6 +510,14 @@ class AsyncRedis {
         return _command<OptionalStringPair>(fmt::blpop, key, timeout);
     }
 
+    template <typename Callback>
+    auto blpop(const StringView &key,
+               Callback &&cb,
```

Comment:
Can you make `cb` be the last parameter? So that the prototype of `blpop`s are consistent with other methods. In order to have default timeout, we can define two `blpop` methods with callback (Please also modify other methods with default parameters, such as `brpop`, `brpoplpush`, `zadd`, and so on):

```
// Async method with timeout as parameter
template <typename Callback>
auto blpop(const StringView &key, const std::chrono::seconds &timeout, Callback &&cb)...

// Async method with a default timeout parameter.
template <typename Callback>
auto blpop(const StringView &key, Callback &&cb) {
    blpop(key, std::chrono::seconds(0), std::forward<Callback>(cb));
}
```